source/dnode/mnode/impl/src/mndMain.c

```diff
@@ -796,6 +797,12 @@ int32_t mndStart(SMnode *pMnode) {
       return -1;
     }
     mndSetRestored(pMnode, true);
+  } else {
```

Comment:
```suggestion
  } else {
```
为什么要在deploy的else里添加逻辑？

Reply:
deploy 是个预留的钩子，会在集群首次部署时执行。但是集群重启没有类似的钩子，所以我增加了一个。